sigstore-java/src/main/java/dev/sigstore/oidc/client/TokenStringOidcClient.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore.oidc.client;
+
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.client.json.webtoken.JsonWebSignature;
+import java.io.IOException;
+
```

Comment:
A comment up here would be nice, perhaps just describing what the string *should* look like, even if it's just a link. If use of this is limited to just conformance, then maybe it should go in the sigstore-cli project instead?

Reply:
Unless I'm misunderstanding, the string in question is the OIDC token, which means that it's shaped like an ordinary JWT. Happy to add a comment to that effect, if you think it'd be useful!

Reply:
uh it's just that it's in the sigstore-java library and hence is available through it's api, so we want to document it. I think it should just go into the sigstore-cli project for now though. I don't see when we would be using this.

Reply:
`TokenStringOidcClient` provokes use of static token (which is fine for testing purposes), however, in practice the tokes will be short-lived, so I doubt static `TokenStringOidcClient` would be useful for sigstore-java users.

It might be a good idea to move `TokenStringOidcClient` to `sigstore-cli` module.

Reply:
[55da240](https://github.com/sigstore/sigstore-java/pull/436/commits/55da240337cff33c74af610607bf379048522df7)